include/openthread/link.h

```diff
@@ -1127,6 +1149,26 @@ otError otLinkSetEnabled(otInstance *aInstance, bool aEnable);
  */
 bool otLinkIsEnabled(otInstance *aInstance);
 
+/**
+ * This function indicates whether or not the link layer is csl enabled.
+ *
+ * @param[in]  aInstance A pointer to an OpenThread instance.
+ *
+ * @retval true   Link layer is csl enabled.
+ * @retval false  Link layer is not csl enabled.
```

Comment:
Use Capital case for true and false to keep consistent.

Reply:
Fixed.